samples/mpsl/timeslot/src/main.c

```diff
@@ -84,10 +106,16 @@ static mpsl_timeslot_signal_return_param_t *mpsl_timeslot_callback(
 			signal_callback_return_param.params.request.p_next =
```

Comment:
The comment one line above is not valid anymore

Reply:
updated